src/main/java/frc/robot/hardware/phoenix6/gyro/Pigeon2Gyro.java

```diff
@@ -19,4 +20,9 @@ public void setYaw(Rotation2d yaw) {
 		gyro.setYaw(yaw);
 	}
 
+	@Override
+	public Pigeon2 getDevice() {
```

Comment:
why not the wrapper? either return wrapper or parent device, no need to choose a class in between